tutorial/westeros/westeros_firm_capacity.ipynb

```diff
@@ -80,17 +98,20 @@
     "        'time' : 'year',\n",
     "        'value' : 2,\n",
     "        'unit' : '???'})\n",
-    "\n",
     "scen.add_par('peak_load_factor', peak_load_factor)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Describing the Renewable Technologies Reliability\n",
-    "### rating and reliability --> Ensuring enough Firm Capacity\n",
-    "we assume, that up to a share of 20% of total electricity supply, the capacity of wind_ppl adds by 80% to the firm capacity of the power system (r1). The remaining 90% installed capacity, however  only  5% contribute to the firm capcity (r2). The coal power plant supplies full reliability independet of the supply share.\n"
+    "## Rating bins and reliability factors - Describing the powerplant reliability\n",
+    "### Ensuring enough firm capacity\n",
+    "Toward meeting the firm capacity requirement, conventional generating technologies contribute their nameplate generation capacity. \n",
+    "\n",
+    "This therefore implies that for conventional powerplants, a reliability factor of 1 is assumed.\n",
+    "\n",
+    "In order to account for the integration challenge at higher shares of variable renewables, their contribution to the capacity requirements, capacity value, declines as the market share of the technology increases.  We therefore assume, that up to a share of 20% of total electricity supply, the capacity of wind_ppl adds by 80% to the firm capacity of the power system (r1). The remaining 80% installed capacity, however  only  5% contribute to the firm capcity (r2). "
```

Comment:
suggest to replace *, capacity value,* by *(i.e., their capacity value)*

Please use formatting for names of technologies and rating bins, too.